### PR TITLE
[GUI] Fix labels scrolling when set to 'false'

### PR DESCRIPTION
**Pull request by Hitcher**
## Description

Reverts the removal of the 'false' bool for a labels scrolling control

## Motivation and Context

Solves the problem of labels scrolling in lists even when using the `false` flag that was introduced with PR #12213

## How Has This Been Tested?

Yes, tested both false and true bools.

## Types of change

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/guilib/GUIListLabel.cpp [hunk 1]
```diff
@@ -29,7 +29,10 @@ CGUIListLabel::~CGUIListLabel(void) = default;
 
 void CGUIListLabel::SetScrolling(bool scrolling)
 {
-  m_label.SetScrolling(scrolling);
+  if (m_scroll == CGUIControl::FOCUS)
+    m_label.SetScrolling(scrolling);
+  else
+    m_label.SetScrolling((m_scroll == CGUIControl::ALWAYS) ? true : false);
```
Comment (ksooo) [inline xbmc/guilib/GUIListLabel.cpp]: This can be simplified: `m_label.SetScrolling(m_scroll == CGUIControl::ALWAYS);` does exactly the same.

**Review by ksooo — COMMENTED** [review]
Just a coding minor. Cannot say anything to the functional change, though.

**Comment by pkerling** [comment]
IMO switch/case would be nicest here

**Comment by HitcherUK** [comment]
If someone has a better method then please submit a new PR and I'll close this one but all I've done is change the code back to how it was before PR #12213 to fix the issue. I don't understand the actual code involved.

**Comment by HitcherUK** [comment]
Superseded by #https://github.com/xbmc/xbmc/pull/15124

**Pull request closed without merging**